platform/javascript/js/libs/audio.worklet.js

```diff
@@ -115,7 +115,7 @@ class GodotProcessor extends AudioWorkletProcessor {
 			this.input = new RingBuffer(p_data[1], avail_in);
 			this.output = new RingBuffer(p_data[2], avail_out);
 		} else if (p_cmd === 'stop') {
-			this.runing = false;
+			this.running = false;
```

Comment:
I don't see any other changed occurrence nor any other match, so this might be unused. Poke @Faless.

Reply:
@akien-mga see line 129
```
	process(inputs, outputs, parameters) {
		if (!this.running) {
			return false; // Stop processing.
```
https://github.com/luzpaz/godot/blob/70bc49e13d90959f7efbfec0a5ddc6a4a8959ce2/platform/javascript/js/libs/audio.worklet.js#L129

Reply:
Thanks, that's actually a bugfix then (or will create a bug, we'll see :D).

Reply:
Please advise how to proceed?

Reply:
It's fine as is.

Reply:
I've checked all mentions of _worklet_s in issues and pull requests in this repository, and don't think there will be any conflicts. The misspelling leads me to believe that a feature was missing, and this feature seems reasonable, so I see no reason not to push unless I have missed something.